### ISSUE TITLE
Speaker remains in group although it was ungrouped

### ISSUE THREAD

**Issue by koen-php** [duncan3dc/sonos#99]
I have the following situation: 
- Speakers were grouped via the Sonos application (Cuisine and Salle-a-manger)
- After that they are ungrouped and the playlists are emptied
- Launching a simple test script afterwards (so new PHP script, without cache) with the Sonos library, still shows the speaker as part of the group (Salle-a-manger has Cuisine UUID as group)
- Result is that sending a command to the zone (Salle-a-manger) doesn't work, because the command is forwarded to the  previous 'group controller' (Cuisine in the example)

```
$sonos = new \duncan3dc\Sonos\Network($devices);
$speakers = $sonos->getSpeakers();
foreach ($speakers as $speaker) {
    /* @var duncan3dc\Sonos\Speaker $speaker */
    echo "ROOM: " . $speaker->getRoom() . "\n";
    echo "\tName: " . $speaker->getName() . "\n";
    echo "\tIP: " . $speaker->getIp() . "\n";
    echo "\tCoordinator: " . $speaker->isCoordinator() . "\n";
    echo "\tUUID: " . $speaker->getUuid() . "\n";
    echo "\tGroup: " . $speaker->getGroup() . "\n";
}
```

![20181119_093915_resized](https://user-images.githubusercontent.com/42060847/48696123-4efa5e00-ebe1-11e8-819d-3150304460f0.jpg)

**Comment by duncan3dc** [comment]
Hi @koen-php, I've managed to reproduce this issue, I'm just investigating it now

**Comment by duncan3dc** [comment]
This is fixed in [2.0.5](https://github.com/duncan3dc/sonos/releases/tag/2.0.5)
